### PR TITLE
Send missing string for translation

### PR DESCRIPTION
**Pull request by CrisBarreiro**
Task/Issue URL: 

### Description
Note: Estonian Strings are missing due to a delay in translations. Created https://app.asana.com/0/0/1208558284050278/f as a follow up to add them when ready

### Steps to test this PR

_Feature 1_
- [ ]
- [ ]

### UI changes
| Before  | After |
| ------ | ----- |
!(Upload before screenshot)|(Upload after screenshot)|

### CODE REVIEW

**Comment by CrisBarreiro** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5146?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5146** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5146?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#5145** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5145?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5136** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5136?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5125** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5125?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5124** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5124?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5109** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5109?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2 other dependent PRs ([#5110](https://github.com/duckduckgo/Android/pull/5110) <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5110?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#5121](https://github.com/duckduckgo/Android/pull/5121) <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5121?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CrisBarreiro and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: duckplayer/duckplayer-impl/src/main/res/values/strings-duckplayer.xml [hunk 1]
```diff
@@ -33,6 +33,8 @@
     <string name="duck_player_unavailable">Duck Player Unavailable</string>
     <string name="duck_player_unavailable_description">Duck Player\'s functionality has been affected by recent changes to YouTube. We’re working to fix these issues and appreciate your understanding.</string>
     <string name="duck_player_unavailable_learn_more">Learn More</string>
+    <!-- TODO: Remove this ignore when the translation is added -->
+    <string name="duck_player_open_videos_setting" tools:ignore="MissingTranslation">Open YouTube Videos in Duck Player</string>
     <!-- TODO(cbarreiro): Add missing translations asap -->
     <string name="duck_player_open_in_a_new_tab_setting" tools:ignore="MissingTranslation">Open Duck Player in a new tab</string>
```
Comment (joshliebe) [inline duckplayer/duckplayer-impl/src/main/res/values/strings-duckplayer.xml:39]: Are both of these ignores needed?

Comment (CrisBarreiro) [reply duckplayer/duckplayer-impl/src/main/res/values/strings-duckplayer.xml:39]: Yes, both translations are missing in Estonian

Comment (joshliebe) [reply duckplayer/duckplayer-impl/src/main/res/values/strings-duckplayer.xml:39]: Follow-up task: https://app.asana.com/0/0/1208558284050278/f

**Review by joshliebe — APPROVED** [review]

**Pull request merged**